sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -785,6 +785,14 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(false)
 
+  val ALLOW_DISABLE_ENTIRE_PLAN = conf("spark.rapids.allowDisableEntirePlan")
+    .internal()
+    .doc("Allow the disabling of all GPU operations for the entire plan when " +
+      "certain conditions happen. Change this to false will revert the behavior to check " +
```

Comment:
This is very vague and confusing.  Perhaps we should reword it to say something like
"The plugin has the ability to detect possibe incompatibility with some specific queries and cluster configurations. In those cases the plugin will disable GPU support for the entire query. Set this to false if you want to override that behavior, but use with caution."

Reply:
updated, thanks, I struggled a bit with this so happy to update to something that makes more sense to others.